test/endpoints/game.spec.js

```diff
@@ -109,6 +124,45 @@ describe('Game', () => {
       });
   });
 
+  it('should return a list of previous games played', (done) => {
+    request.get('/api/games/history')
+      .set('authorization', user.token)
+      .expect(200)
+      .end((err, res) => {
+        if (err) {
+          return done(err);
+        }
+        (Array.isArray(res.body)).should.be.eql(true);
```

Comment:
@Daramola98 What if it's an empty array? Is that valid too?

Reply:
yes because it is possible that no one has donated on the platform

Reply:
Refactored @fortunee